core/src/main/java/io/questdb/griffin/engine/functions/math/LeastNumericFunctionFactory.java

```diff
@@ -81,16 +83,20 @@ public Function newInstance(
                 case ColumnType.BYTE:
                 case ColumnType.DATE:
                 case ColumnType.TIMESTAMP:
+                    counters[type]++;
```

Comment:
`counters[type]++` will execute twice here because of the (unwanted?) fall-through to the `case ColumnType.NULL` branch.